content/it/_index.md

```diff
@@ -0,0 +1,25 @@
+
+---
+title: "Glossario Cloud Native"
+---
+
+# Glossario Cloud Native
+
+Il Glossario Cloud Native è un progetto guidato dal Business Value Subcommittee (BVS) della CNCF con cui la community si propone di spiegare concetti del mondo cloud native con un linguaggio semplice e diretto che non presupponga conoscenze tecniche pregresse. La [versione in PDF può essere visualizzata e scaricata qui] (https://github.com/cncf/glossary/blob/main/cloudnative-glossary.pdf).
+
+
+## Contribuire
+
+L'invito a suggerire modifiche, aggiunte e migliorie al Glossario Cloud Native è aperto a chiunque abbia voglia di contribuire. Il processo che adottiamo per lo sviluppo e l'arricchimento di questo dizionario condiviso è guidato dalla community e governato dalla CNCF. Il Glossario fornisce una piattaforma vendor-neutral per organizzare un vocabolario condiviso riguardante le tecnologie cloud native. 
+Chiunque voglia contribuire può aprire una issue o una pull request su GitHub. Leggi la sezione [Come Contribuire](/contribuire/) per maggiori informazioni e assicurati di seguire le indicazioni della [Style Guide](/style-guide/).
+
+
+## Riconoscimenti
+
+Il progetto del Glossario Cloud Native è un'iniziativa del Marketing Committee della CNCF (nello specifico del Business Value Subcommittee) e include contributi da [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/) e [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca). 
```

Comment:
There has been an update here. Seokho and the list of all contributors is missing

> The Cloud Native Glossary was initiated by the CNCF Marketing Committee (Business Value Subcommittee) and includes contributions from [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) and many more contributors. For a complete contributor list, please refer to [this GitHub page](https://github.com/cncf/glossary/graphs/contributors).

Reply:
done